index.html

```diff
@@ -145,6 +145,25 @@ <h5>About me</h5>
             </ul>
         </div>
     </section>
-    <footer></footer>
+    <footer>
+        <h1>I'm always <br> interested in hearing about new projects, so if you'd like to chat please get in touch.</h1>
+        <form action="https://formspree.io/f/{form_id}" method="post"> 
```

Comment:
- [ ] Well done using the post method on the form, but you are required to sign up on the [formspree service](https://formspree.io/html) and integrate your form using the form_id, Kindly fix it , When I submit the form it show me the following message.

![Capture](https://user-images.githubusercontent.com/93598189/205949646-9e30a750-b7fb-40a3-a184-c2277a316106.PNG)